cybershell/continuous_learning_pipeline.py

```
@@ -0,0 +1,424 @@
+                        learning_rate=0.1,
+                        max_depth=5,
+                        random_state=42
+                    )
+                else:
+                    self.models[model_name] = RandomForestClassifier(
+                        n_estimators=50,
+                        random_state=42
+                    )
+                
+                self.scalers[model_name] = StandardScaler()
+                logger.info(f"Initialized new model: {model_name}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)